### DISCUSSION TITLE
FileInput

### DISCUSSION THREAD

**Discussion by mh3i** [mantinedev/mantine#6605] [Q&A] [answered]
how can I limit File input to accept minimum of two files 
source :
import { FileInput } from '@mantine/core';

function Demo() {
  return <FileInput label="Upload files" placeholder="Upload files" multiple />;
}

**Comment by rtivital** [comment, marked as answer]
As far as I know, `input[type="file"]` does not support such feature

**Reply by mh3i** [reply]
Got it. thank you !
